ansys/dpf/core/available_result.py

```diff
@@ -53,25 +54,26 @@ def __str__(self):
 
     @property
     def name(self):
-        """Result operator"""
+        """Result operator."""
         if self.operator_name in _result_properties:
             return _result_properties[self.operator_name]["scripting_name"]
         else:
             return _remove_spaces(self._message.physicsname)
 
     @property
     def n_components(self):
-        """Number of components of the results"""
+        """Number of components of the result."""
         return self._message.ncomp
 
     @property
     def dimensionality(self):
-        """Dimensionality nature of the result (vector, scalar, tensor...)"""
+        """Dimensionality nature of the result, such as a vector, scalar, tensor, or 
+        so on."""
```

Comment:
Let's avoid "or so on" and use "etc."

Also this should be on one line if we can manage it.  Longer descriptions should include a long description section.

Reply:
@akaszynski  Ansys (and all other tech doc style guides) say no Latin words like etc., i.e., ....